test/alphabet/alphabet_test.cpp

```diff
@@ -33,10 +33,20 @@
 // ==========================================================================
 
 #include <gtest/gtest.h>
-#include <sstream>
 
 #include <seqan3/alphabet/all.hpp>
 
+#if SEQAN_WITH_CEREAL
```

Comment:
> again check directly with #if __has_include() it's more easy to understand than some self written macros.

1) see above, we need to be able to deactivate, even if its there
2) it would be much harder to maintain. Imagine they rename the top-leve file, then we have to search through all of our code-base to change this. Right now we need to change the macro and a single include in core.
3) Also imagine there is cereal on disk, but it's the wrong version. A macro can have multiple conditions, adding multiple conditions to every file will get really complicated.

> And this are our test cases so it makes no sense that a user may disable them. For the library itself it might be sensible to be able to disable it, but the test cases are our domain.

But the test cases are supposed to ensure it works for our users → we have to test if our library works without cereal, because our users might not have it!!